guides/common/modules/ref_system-requirements.adoc

```diff
@@ -41,6 +41,10 @@ endif::[]
 * A system umask of 0022
 * Full forward and reverse DNS resolution using a fully-qualified domain name
 
+The {Project} and {SmartProxy} server do not support shortnames in the hostnames.
+Also, the custom certificate's Common Name (CN) must have a fully qualified domain name (FQDN) instead of a shortname.
+This does not apply to the clients of a {Project}.
```

Comment:
I know the goal of this statement is to head off getting this question, which we get every time we talk about shortname support. Users hear about this and ask "does this also apply to clients", so while it seems obvious it should not since we did not statement I am inclined towards keeping this statement to head off unnecessary back and forth.